aviary/utils/fortran_to_aviary.py

```diff
@@ -383,6 +383,9 @@ def update_gasp_options(vehicle_data):
     """
     input_values: NamedValues = vehicle_data['input_values']
 
+    for var_name in gasp_scalar_variables:
```

Comment:
Note:

scalar = non-vector quantity
scaler = factor used to scale a quantity (multiplicative in this case)

Everywhere here that is "scalar" should be "scaler".